Platform.XamarinIOS/Realm.Sync.XamarinIOS/Realm.Sync.XamarinIOS.csproj

```diff
@@ -69,6 +69,7 @@
     <Reference Include="Xamarin.iOS" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Json" />
+    <Reference Include="System.Core" />
```

Comment:
I'm not sure why, but without that, the LINQ expression here: https://github.com/realm/realm-dotnet/pull/973/files#diff-6842da5be1f20518257e6c13d8e1e763R143 failed to compile.